Gemfile.lock

```diff
@@ -143,7 +143,7 @@ GEM
     config (2.2.3)
       deep_merge (~> 1.2, >= 1.2.1)
       dry-validation (~> 1.0, >= 1.0.0)
-    connection_pool (2.2.2)
+    connection_pool (2.5.3)
```

Comment:
Also, where is this used and can we rip it out?

Reply:
That's a good question. I'll need to investigate more.

Reply:
It's a [requirement](https://github.com/rails/rails/blob/3235827585d87661942c91bc81f64f56d710f0b2/activesupport/lib/active_support/cache/mem_cache_store.rb#L10) for using the [`:mem_cache_store`](https://api.rubyonrails.org/classes/ActiveSupport/Cache/MemCacheStore.html) as a cache store.

Reply:
I noticed we are also deployed with Redis available. Is there any reason we couldn't migrate off MemCache and consolidate into Redis entirely?

Reply:
I would prefer to depend less on Redis given in general since they're proprietary now. (Although Valkey seems like a reasonable alternative)

Originally the cache and sessions were on Memcached. The main issues with memcached was there was no way to backup sessions. People would get logged out if you ever needed to change memcached providers or the memcached version needed to be updated. We moved to cookies but for security reasons, we wanted sessions that were properly ended after a log out. Cookies don't do that so redis was the option.

Nowadays, I'd probably consider using SolidCache and ActiveRecord SessionStore; that said there's issues with the size of our database to consider that might make that unwise.